### PR TITLE
update BUAN citations

### PR DESCRIPTION
**Pull request by BramshQamar**
This PR updates BUAN paper citations.

### CODE REVIEW

File: dipy/segment/bundles.py [hunk 1]
```diff
@@ -178,6 +178,11 @@ def bundle_shape_similarity(bundle1, bundle2, rng, clust_thr=[5, 3, 1.5],
 
     References
     ----------
+    .. [Chandio2020] Chandio, B.Q., Risacher, S.L., Pestilli, F. et al.
```
Comment (skoudoro) [inline dipy/segment/bundles.py:181]: Can you add all authors? thank you

File: dipy/segment/bundles.py [hunk 2]
```diff
@@ -459,6 +464,11 @@ def refine(self, model_bundle, pruned_streamlines, model_clust_thr,
         .. [Garyfallidis17] Garyfallidis et al. Recognition of white matter
             bundles using local and global streamline-based registration and
             clustering, Neuroimage, 2017.
+
+        .. [Chandio2020] Chandio, B.Q., Risacher, S.L., Pestilli, F. et al.
```
Comment (skoudoro) [inline dipy/segment/bundles.py:468]: Can you add all authors? thank you

**Review by skoudoro — COMMENTED** [review]
Hi @BramshQamar,

Thank you for this update. Overall, I think it will be good to add all authors to your citation. Let me know what you think. Thank you.

**Comment by BramshQamar** [comment]
> Hi @BramshQamar,
> 
> Thank you for this update. Overall, I think it will be good to add all authors to your citation. Let me know what you think. Thank you.

Thank you for the feedback @skoudoro. I have updated the PR with all authors' names in the citation.

**Comment by arokem** [comment]
Looks great! Could you please reflow the lines so as to address the PEP8 issues? I think that D.Bullock's name is sticking out beyond the 80 character limit.

**Comment by BramshQamar** [comment]
> Looks great! Could you please reflow the lines so as to address the PEP8 issues? I think that D.Bullock's name is sticking out beyond the 80 character limit.

Fixed PEP8 errors. Thank you!

**Comment by skoudoro** [comment]
Great! thank you @BramshQamar! Not waiting for the CI's, merging!

**Pull request merged**